services/web/server/src/simcore_service_webserver/projects/projects_api.py

```diff
@@ -1414,4 +1414,4 @@ async def get_project_inactivity(
     project_inactivity = await director_v2_api.get_project_inactivity(
         app, project_id, max_inactivity_seconds
     )
-    return parse_obj_as(Envelope[GetProjectInactivityResponse], project_inactivity)
```

Comment:
yep, so the directorv2 is returning a non-enveloped response and here you are creating an envelope for the webserver, right?
https://github.com/ITISFoundation/osparc-simcore/blob/master/services/director-v2/openapi.json#L2142-L2153

What i wonder is why some of the responses in the directorv2 are enveloped and why others not  :-(

Reply:
Actually there is only 1 endpoint which is enveloped in director-v2 and I think it's a mistake.
The pattern appears clear. Director-v2 does not envelope responses

Reply:
I see. Yes, it seems only `RetrieveDataOutEnveloped` is enveloped. I gues that was the confusion in this bug.
I suggest to create an issue to remind converting that entry from enveloped to non-evenloped